src/routes/safe/components/Balances/SendModal/index.tsx

```diff
@@ -74,6 +74,7 @@ const SendModal = ({
   const [activeScreen, setActiveScreen] = useState<TxType>(activeScreenType || 'chooseTxType')
   const [tx, setTx] = useState<unknown>({})
   const [isABI, setIsABI] = useState(true)
+  const [recipient, setRecipient] = useState(recipientAddress)
```

Comment:
Please add a type for this state.

Reply:
This will work for the initial rendering but not if the prop changes, am I right?

Reply:
Yes, you're right. I'll add the hook to update when the props change.